### DISCUSSION TITLE
OSX: how to make Macvim available as a system editor

### DISCUSSION THREAD

**Discussion by BackSeat** [Homebrew/discussions#3346] [Everyday usage] [answered]
I have Macvim installed on OSX. When OSX wants to open a text file, it offers me a list of programs that can help. Macvim isn't one of them, presumably because it's installed under /opt rather than in Applications.

What's the best way of fixing that?

**Comment by hannesbraun** [comment, marked as answer]
I assume you installed MacVim as a formula (`brew install macvim`). That would be the reason for MacVim being installed in `/opt`. To be able to open text files from outside the Terminal with MacVim, you probably want to rather install the cask (`brew install --cask macvim`). Then MacVim should appear when you right click on a text file for example.

Hope that helps.

**Reply by BackSeat** [reply]
Fixed, thank you so much!
